src/components/reviews/Reviews.js

```diff
@@ -0,0 +1,63 @@
+import { Review } from "./Review";
+import { useState, useEffect } from "react";
+import styles from "@/styles/reviews.module.scss";
+import { useContext } from "react";
+import { LoggedInContext } from "@/pages/_app";
+
+export const Reviews = (props) => {
+  const { handleShowLogInModal, isLoggedIn } = useContext(LoggedInContext);
+  const [reviews, setReviews] = useState([]);
+  const [once, setOnce] = useState(false);
+
+  const handleLogin = () => {
+    window.scrollTo(0, 0);
+    handleShowLogInModal();
+  };
+  if (props.posted && once == true) {
+    setOnce(false);
+  }
+  useEffect(() => {
+    if (!once) {
+      const movieId = window.location.pathname.split("/")[2];
+      getData(movieId);
+      props.setPosted(false);
+      setOnce(true);
+    }
+    async function getData(movieId) {
+      const res = await fetch("/api/movies/" + movieId + "/reviews");
+      if (!res.ok) {
+        throw new Error("Failed to fetch data");
+      }
+      const reviews = await res.json();
+      setReviews(
+        reviews.map(
```

Comment:
You should probably store the review _data_ in state instead, and then loop over it to render on line 61.